app/src/main/java/com/nervousfish/nervousfish/exceptions/NoBluetoothException.java

```diff
@@ -0,0 +1,24 @@
+package com.nervousfish.nervousfish.exceptions;
+
+/**
+ * Simple {@link RuntimeException} that is used to notify usage of Bluetooth is not avaiable.
+ */
+public class NoBluetoothException extends RuntimeException {
```

Comment:
Go to Settings -> Editor -> Inspections -> Java and enable everything from Class metrics, from Code style issues "Instance field access not qualified with this, Instance method not qualified with this", from Inheritance issues Missing Override annotation, everything from concurrency and for this line of code specifically: everything from serialization. When you hover over the classname our brilliant idea will hint you to generate a serialversionuid and nice readobject/writeobject methods.